Source/TailwindTraders.ShippingManagement/Services/ResponseService.cs

```diff
@@ -74,66 +74,61 @@ private WrapForm ReadJson()
                 {
-                    var kv = page.KeyValuePairs.Where(c => c.Key[header.KeyPos].Text == header.Key).First();
+                    var kv = page.Fields.Where(c => c.Key[header.KeyPos].ToString() == header.Key).First();
 
-                    if (kv.Value.Any())
+                    if (kv.Value != null)
                     {
                         switch (header.Type)
                         {
                             case "int":
-                                _model.SetProperty(header.Property, int.TryParse(kv.Value[header.KeyValue].Text, out int n) ? n : default);
+                                _model.SetProperty(header.Property, int.TryParse(kv.Value.ValueData.Text, out int n) ? n : default);
                                 break;
                             case "decimal":
-                                _model.SetProperty(header.Property, decimal.TryParse(kv.Value[header.KeyValue].Text, out decimal d) ? d : default);
+                                _model.SetProperty(header.Property, decimal.TryParse(kv.Value.ValueData.Text, out decimal d) ? d : default);
                                 break;
                             case "string":
                             default:
-                                _model.SetProperty(header.Property, kv.Value[header.KeyValue].Text);
+                                _model.SetProperty(header.Property, kv.Value.ValueData.Text);
                                 break;
                         }
                     }
                 }
             }
         }
 
-        private List<Item> ParseItems(ExtractedPage page, int numTable = 0)
+        private List<Item> ParseItems(RecognizedForm page, int numTable = 0)
         {
             List<Item> items = new List<Item>();
 
-            foreach (var t in page.Tables)
+            foreach (var t in page.Pages.First().Tables)
             {
                 ProvisioningItems(t, items);
 
-                foreach (var c in t.Columns)
-                {
+                foreach (var c in t.Cells) {
                     var index = 0;
-                    foreach (var e in c.Entries)
-                    {
-                        Item item = items[index];
-                        item.ID = index;
+                    Item item = items[index];
+                    item.ID = index;
 
-                        foreach (var ee in e)
+                        ItemProperty itemProp = new ItemProperty()
                         {
-                            ItemProperty itemProp = new ItemProperty()
-                            {
-                                Value = ee.Text,
-                                Accuracy = ee.Confidence.HasValue ? (double)ee.Confidence : 0
-                            };
+                            Value = c.Text,
+                            Accuracy = String.IsNullOrEmpty(c.Confidence.ToString()) ? (double)c.Confidence : 0
```

Comment:
are you finding that the `c.Confidence` property is returning empty? or other default value?
I haven't seen this and looking at the code, there will always be a value here... so I am curious.